From 42e24022d09dfe7c1a0a4bee25a3b7130e72e116 Mon Sep 17 00:00:00 2001
From: Nicolas Ferre <nicolas.ferre@atmel.com>
Date: Mon, 8 Oct 2012 16:55:29 +0200
Subject: [PATCH 019/397] MTD: atmel_nand: add 9n12 to list of SoC with DMA

Temporary: may have to be replaced by a device-tree property.

Signed-off-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 drivers/mtd/nand/atmel_nand.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/atmel_nand.c b/drivers/mtd/nand/atmel_nand.c
index aca8108..58bb70a 100644
--- a/drivers/mtd/nand/atmel_nand.c
+++ b/drivers/mtd/nand/atmel_nand.c
@@ -127,7 +127,8 @@ static struct nand_ecclayout atmel_pmecc_oobinfo;
 
 static int cpu_has_dma(void)
 {
-	return cpu_is_at91sam9rl() || cpu_is_at91sam9g45() || cpu_is_at91sam9x5();
+	return cpu_is_at91sam9rl() || cpu_is_at91sam9g45()
+		|| cpu_is_at91sam9x5() || cpu_is_at91sam9n12();
 }
 
 /*
-- 
1.8.5.3

